### ISSUE TITLE
-m options was useful

### ISSUE THREAD

**Issue by andreaganduglia** [kardolus/chatgpt-cli#7]
I've tried the new version and I believe it's important to reinstate the -m option so that the model can be switched during runtime.

To balance the cost of the APIs, it is often useful to operate with GPT-3 and then pass the output to GPT-4, for example:

```
chatgpt -m gpt-3 "Do something" | chatgpt -m gpt-4 "Do something else" | ...
```

Using global variables or file configurations forces one to break the pipeline and save partial results in temporary files, which is not aligned with the rest of the utilities. This doesn't only apply to this scenario, but also when a user wants to change the model on the fly, or create a command alias.

Before this version, I had some very useful aliases:

```
alias gpt4="chatgpt -m gpt-4"
alias gpt3="chatgpt -m gpt-3-turbo"
```

which can no longer be created now.

In summary, I believe that:

- every option should have a default value
- every option present in the yaml file should have a corresponding argument
- the hierarchy should be: default, config, argument
- local variables should be used only for API_KEY

I hope these considerations may be of help!
Thank you!

**Comment by kardolus** [comment]
Thanks for your feedback! Do you think this would work for you as well? 

default, config, environment, argument.

I rather keep the current environment variables. I can add all the command line flags (ie. -m). 

Note that you can do this:
```
alias gpt4="OPENAI_MODEL=gpt-4 chatgpt"
```

This will have the same effect as setting the command line flag.
